VERSION

```diff
@@ -1 +1 @@
-7.3.1
+7.4.0
```

Comment:
Since we are adding an example we don't need to change the version IMO, maybe @souza-dan thinks otherwise.

Reply:
That makes sense, the module itself hasn't changed.

Reply:
Version 7.3.2 will be populated then. Thank you,